.circleci/config.yml

```diff
@@ -77,6 +77,7 @@ jobs:
             - v4-dependencies-
       - run: yarn install --pure-lockfile
       - run: yarn lint:check
+      - run: yarn workspace @emotion/serialize test:typescript
```

Comment:
not running `test:typescript` has already been fixed on the `next` branch, I would keep this out from here to keep merging simpler

Reply:
```suggestion
```